docs/minio.md

```diff
@@ -11,341 +11,305 @@ sidebar_label: MinIO
 
-[MinIO](https://github.com/minio/minio) is a high performance distributed object storage server, designed for large-scale private cloud infrastructure. MinIO is designed in a cloud-native manner to scale sustainably in multi-tenant environments. Orchestration platforms like Kubernetes provide a perfect cloud-native environment to deploy and scale MinIO. MinIO can be provisioned with OpenEBS volumes using various OpenEBS storage engines such as [Local PV](https://docs.openebs.io/docs/next/localpv.html), [cStor](https://docs.openebs.io/docs/next/cstor.html), or [Jiva](https://docs.openebs.io/docs/next/jiva.html) based on the application requirement.
+MinIO can be provisioned with OpenEBS volumes using various OpenEBS storage engines such as [Local PV](https://docs.openebs.io/docs/next/localpv.html), [cStor](https://docs.openebs.io/docs/next/cstor.html), or [Jiva](https://docs.openebs.io/docs/next/jiva.html) based on the application requirement. The MinIO operator offers a seamless way to create and update highly available distributed MinIO clusters. MinIO operator brings native support for MinIO, Graphical Console for Admin and Users, and encryption to Kubernetes. It also offers MinIO tenant creation, management, upgrade, zone addition, and more.
 
+Depending on the performance and high availability requirements of MinIO, you can select any of the storage engine to run MinIO with the following deployment options:
+- For optimal performance, deploy MinIO with OpenEBS Local PV. 
+- If you would like to use storage layer capabilities like high availability, snapshots, incremental backups and restore and so forth, you can select OpenEBS cStor.
 
-
-**Advantages of using OpenEBS underneath MinIO Object Storage:**
-
-- PVCs to MinIO are dynamically provisioned from a dedicated or shared storage pool. Therefore, the requirement of dedicated disks on bare-metalserver or on a cloud environment to launch an object storage solution can be avoided.
-- Adding more storage to the Kubernetes cluster with OpenEBS is seamless and done along with adding a Kubernetes node. The auto-scaling of MinIO instances on-premises is as easy as cloud platforms.
-- Storage within a node or pool instance can also be scaled up on demand. 
-- Complete management of disks under MinIO is managed by OpenEBS, leading to production-grade deployment of object storage.
-
-<br>
-
+This document provides the instructions to setup MinIO operator using OpenEBS Local PV.
 
 
 ## Deployment model
 
-<br>
-
-<img src="/docs/assets/minio-choose-engine.png" alt="OpenEBS and MinIO" style="width:400px;">
-
-<br>
-
+<img src="/docs/assets/svg/Local-PV-Distributed-device-minio.svg" alt="OpenEBS and MinIO Distributed localpv device" style="width:400px;">
```

Comment:
Can the image's width and height be increased, so that it is readable. The current dimension does not provides clarity(fonts and images are too small) in the content shown in the diagram.

Reply:
Changed width as "width:100%; from "width:400px;"  
Don't have much knowledge about this. Just doing one trial.

Reply:
Looks better now